sw/device/tests/csrng_kat_test.c

```diff
@@ -14,19 +14,19 @@
 
 OTTF_DEFINE_TEST_CONFIG();
 
-void test_ctr_drbg_ctr0(const dif_csrng_t *csrng) {
-  CHECK_DIF_OK(dif_csrng_uninstantiate(csrng));
-  csrng_testutils_fips_instantiate_kat(csrng, /*fail_expected=*/false);
-  csrng_testutils_fips_generate_kat(csrng);
+status_t test_ctr_drbg_ctr0(const dif_csrng_t *csrng) {
+  TRY(dif_csrng_uninstantiate(csrng));
+  TRY(csrng_testutils_fips_instantiate_kat(csrng, /*fail_expected=*/false));
+  TRY(csrng_testutils_fips_generate_kat(csrng));
+  dif_result_t res = kDifOk;
+  return INTO_STATUS(res);
 }
 
-bool test_main(void) {
+status_t test_main_status(void) {
   dif_csrng_t csrng;
-  CHECK_DIF_OK(dif_csrng_init(
-      mmio_region_from_addr(TOP_EARLGREY_CSRNG_BASE_ADDR), &csrng));
-  CHECK_DIF_OK(dif_csrng_configure(&csrng));
+  mmio_region_t base_addr = mmio_region_from_addr(TOP_EARLGREY_CSRNG_BASE_ADDR);
+  TRY(dif_csrng_init(base_addr, &csrng));
```

Comment:
you can still use `CHECK_DIF_OK` here.

Reply:
we are just trying to update the testutils, not the DIF calls in every test. We are going to write separate tests for manufacturing that can call DIFs in a different way.

Reply:
done